invesalius/data/visualization/mep_visualizer.py

```diff
@@ -0,0 +1,470 @@
+import random
```

Comment:
Please, add the invesalius header 
# --------------------------------------------------------------------------
# Software:     InVesalius - Software de Reconstrucao 3D de Imagens Medicas
....